assignments/array-methods.js

```diff
@@ -56,28 +56,67 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach(function(currentValue) {
+	fullName.push(`${currentValue.first_name} ${currentValue.last_name}`);
+});
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+const firstNameCaps = runners.map(function(currentValue) {
+	allCaps.push(currentValue.first_name.toUpperCase());
+	return allCaps;
+});
 console.log(allCaps); 
 
 // ==== Challenge 3: Use .filter() ====
 // The large shirts won't be available for the event due to an ordering issue.  Get a list of runners with large sized shirts so they can choose a different size. Return an array named largeShirts that contains information about the runners that have a shirt size of L and log the result
 let largeShirts = [];
+const needNewSize = runners.filter(function(currentValue) {
+	if (currentValue.shirt_size === "L") {
+		largeShirts.push(currentValue);
+	}
+	return largeShirts;
+});
 console.log(largeShirts);
```

Comment:
This is great and it's doing exactly what it's supposed to do, but here's a way to do it in half the lines:

let largeShirts = runners.filter((shirts) => {
  return shirts.shirt_size === "L";
});
console.log(largeShirts);